android/src/main/java/com/thebluealliance/androidclient/helpers/MatchHelper.java

```diff
@@ -593,14 +609,24 @@ public static String getMatchTitleFromMatchKey(String matchKey) {
             }
 
             if(set == null) {
-                // No set specified; this is a quals match
-                return LONG_TYPES.get(TYPE.fromShortType(type)) + " " + number;
+                // No set specified; this is a match like "Quals 10" (abbrev "Q10")
+                String format = abbrev ? "%1$s%2$s" : "%1$s %2$s";
```

Comment:
Agreed.

How do you make static methods get a Context needed to look up resources?

[One trick](http://stackoverflow.com/a/5114361/1682419) is to have the Application instance save it in a static variable, but is that reliable for use within a BroadcastReceiver?